go.mod

```diff
@@ -285,4 +284,4 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
-replace github.com/cert-manager/cert-manager => github.com/openshift/jetstack-cert-manager v1.14.5-0.20240425093532-6a0915206f61
+//replace github.com/cert-manager/cert-manager => github.com/openshift/jetstack-cert-manager v1.14.5-0.20240425093532-6a0915206f61
```

Comment:
The replace directive can be updated to refer to 1.15.2 post the downstream rebase.